### PR TITLE
RHApp Release - Beta Improvements

### PR DESCRIPTION
**Pull request by marcus-ong-qy**
Updates to Frontend code based on beta feedbacks

1. Changed Booking Form Placeholder Text
image

2. Fixed Scroll bug
Time Selectors and Displays will now scroll to the current time, and, if the date selected is not today, scrolls to the default time (currently set at 1800)
image
image

3. Updated Conflict Bookings popup messages
Clearer messages, and distinct modals for single- and recurring booking conflicts

Single Booking conflict:
image

Recurring Booking conflict:
image

4. Fixed Recurring Booking exclusive end date

- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in** the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"

### CODE REVIEW

**Review by gunbux — APPROVED** [review]
lgtm!

**Comment by cjunxiang** [comment]
good to split PRs more, into indiv fixes.

rationale: if one thing spoils the whole app, can just roll back one indiv PR.

File: frontend/src/routes/FacilityBooking/ViewConflicts/ConflictBookingModal.tsx [hunk 1]
```diff
@@ -20,16 +21,18 @@ export default function ConflictBookingModal(props: Props) {
...
     dispatch(
       handleCreateNewBooking(
         booking?.facilityID,
         booking?.eventName,
         booking?.startTime ?? null,
         booking?.endTime ?? null,
+        booking?.endDate !== 0,
         booking?.endDate,
         booking?.ccaID,
         booking?.description,
```
Comment (cjunxiang) [inline frontend/src/routes/FacilityBooking/ViewConflicts/ConflictBookingModal.tsx:38]: no context comment:
isit easier to pass the whole booking object and do the truefalse checks in the action?

Comment (gunbux) [reply frontend/src/routes/FacilityBooking/ViewConflicts/ConflictBookingModal.tsx:38]: > good to split PRs more, into indiv fixes.
> 
> rationale: if one thing spoils the whole app, can just roll back one indiv PR.

re to this: yes, from the earlier releases, resolving merge conflicts from a chain of 200 commits is  not very fun and has a lot of errors as well

**Pull request merged**